src/style.css

```diff
@@ -1 +1,102 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+  padding: 0;
+  margin: 0;
+  
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main-header {
+  display: none;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
```

Comment:
do not add position absolute you can position them with margin and padding
with `position: absolute;` you loose the flexibility